cvat-canvas/src/typescript/canvasModel.ts

```diff
@@ -352,16 +341,11 @@ export class CanvasModelImpl extends MasterImpl implements CanvasModel {
         this.notify(UpdateReasons.SHAPE_ACTIVATED);
     }
 
-    public rotate(rotation: Rotation, remember: boolean = false): void {
-        if (rotation === Rotation.CLOCKWISE90) {
-            this.data.angle += 90;
-        } else {
-            this.data.angle -= 90;
+    public rotate(rotationAngle: number): void {
+        if (this.data.angle !== rotationAngle) {
+            this.data.angle = rotationAngle;
+            this.fit();
```

Comment:
It's better to check here that angle is multiple 90 and round it if necessary
I understand that you do it in ``cvat-ui``, but the key idea is that the same canvas should work with any other UI without problems